lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -292,7 +292,7 @@ def plot(self, *, show_outline=True, show_labels=True, axes=None, **kwargs):
                 self.seconds,
                 self.position,
                 path_effects=[pe.Stroke(foreground="k", linewidth=linewidth * 2.5)],
-                **kwargs,
+                **{**kwargs, "label": "__no_label__"},
```

Comment:
More of a nitpick, but this would explicitly communicate we're skipping the legend:
```suggestion
                **{**kwargs, "label": "__no_legend__"},
```